Added Troubleshoot documents, return management

Amir-HC opened the pull request:
Updated TOC to include shipping method mapping page
Updated product page in troubleshooting top add code blocks for buttons and pages

Amir-HC committed:
Update Products.md

Update products file to add code blocks for job names and section names.

Amir-HC committed:
Update SUMMARY.md

Updated Table of Contents to add shipping method mapping page

Amir-HC committed:
Create inventory.md

Created Inventory page for Shopify troubleshooting

Amir-HC committed:
Update inventory.md

Rectified minor mistakes

Amir-HC committed:
Create order.md

Added Order section for Shopify troubleshoot page

Amir-HC committed:
Create glossary.md

Created a Glossary page for Troubleshooting docs

Amir-HC committed:
Update and rename glossary.md to GLOSSARY.md

Renamed glossary page to follow the naming convention. of glossaries

Dhiraj1405 commented on troubleshooting/Shopify/inventory.md:
```diff
@@ -0,0 +1,39 @@
+
+**Scenario:** Inventory Not Synced from HotWax Commerce to Shopify
+
+Sometimes retailers may encounter inventory disparities between HotWax Commerce and Shopify Inventory. This discrepancy could result in inventory appearing out of stock in Shopify despite being available in HotWax Commerce.
```
Write the other half as an example.

Dhiraj1405 commented on troubleshooting/Shopify/inventory.md:
```diff
@@ -0,0 +1,39 @@
+1. **Check Job Status:**
+
+   - Navigate to the `HotWax Commerce Job Manager App`.
+   - Locate the job named `Upload recent inventory change`. This job is responsible for updating all inventory changes in Shopify that have occurred in HotWax Commerce since the last time the job ran.
```
As both of these jobs are responsible for inventory changes, we should write in that way and ask user to verify on basis of its use case.

Dhiraj1405 commented on troubleshooting/Shopify/inventory.md:
```diff
@@ -0,0 +1,39 @@
+
+   - In HotWax Commerce, Go to `Products` > `Find Products` > `Product Detail page` > `Product Inventory View` page.
+   - Check inventory configurations, including safety stocks, thresholds, reserved inventory, and excluded facilities.
+   - Verify the `online ATP` of products on the inventory dashboard, if the `online ATP` is 0 then the product will appear out of stock in Shopify.
```
We should make it generic, this is reflecting a specific use case

Dhiraj1405 commented on troubleshooting/Shopify/inventory.md:
```diff
@@ -0,0 +1,39 @@
+2. **Verify Inventory Adjustment History:**
+   - Within the variant's `inventory` section, click on the `adjustment history` button.
+   - Examine historical inventory adjustments. Ensure they reflect recent changes.
+   - If historical changes show old records or there are no inventory records, go to the `HotWax Commerce Job manager App` and run the `Hard Sync` job.
```
Write both jobs specific to use case

Amir-HC committed:
Create GLOSSARY.md

Added Glossary for user manuals

Amir-HC committed:
Update inventory.md

Updated Inventory page to add detailed use case of Hard Syn and Update Recent inventory change job

Amir-HC committed:
Create in-store-returns.md

Created In-store returns page for return management user manual

Amir-HC committed:
Update in-store-returns.md

added Shopify Integration link

Amir-HC committed:
Create import-returns.md

Added Import returns from Shopify page

Amir-HC committed:
Create restock-return.md

Created restock returns page for return management

Amir-HC committed:
Create readme.md

Added Readme for return mnagement

Amir-HC committed:
Updated TOC to include returns

Amir-HC committed:
Update MapLocations.md

Updated Location mapping doc to include detail about ecom inventory

Amir-HC committed:
Update MapLocations.md

Added details in mapping ecom inventory location on shopify to hotwax brokering queue
